indico/modules/rb_new/controllers/backend/rooms.py

```diff
@@ -73,7 +73,7 @@ def _process_args(self):
         if not self.room.is_active:
             raise NotFound
 
-
+sdsd
```

Comment:
:warning: 
```suggestion

```

Reply:
:see_no_evil:

Reply:
As a suggestion, run `git add -p` so you can spot these kind of typos easily when adding your changes before committing.